src/index.ts

```diff
@@ -89,7 +92,21 @@ async function buildDenoLambda(
   try {
     await execa(
       path.join(workPath, 'layer', 'bin', 'deno'),
-      ['bundle', entrypointPath, binPath, ...(unstable ? ['--unstable'] : [])],
+      [
+        'bundle',
+        entrypointPath,
+        binPath,
+        // Boolean
+        ...(unstable ? ['--unstable'] : []),
+        // Strings
+        // Deno Version 0.42.0 will parse as 0.42, 1.0.0 will parse as 1
+        // Quick fix for deno version. In the future a better "version check"
+        // could be built. But for right now this fixes the issue.
+        ...(tsConfig &&
+        (DENO_VERSION === 'latest' || parseFloat(DENO_VERSION) >= 1)
```

Comment:
Nice hack :smile:

Reply:
Should we parse deno version for future case? Since it's using semver format, we could easily parse it to `Major.minor.patch`

Reply:
No, it's good enough for now. We can fix it when it breaks :-)